### ISSUE TITLE
Warns "Warning:  duplicate property or selector, consider merging" for `@require` statements

### ISSUE THREAD

**Issue by awayken** [rossPatton/stylint#36]
When the "duplicates" flag is set to true, multiple `@require` statements are warned as being duplicates even if they require different stylus files. To recreate the issue, create the following files and run the command at the bottom.

base.styl
```stylus
body {
    color: red;
}
```

otherfile.styl
```stylus
p {
    padding: 4em;
}
```

site.styl
```stylus
@require 'base';
@require 'otherfile';
```

config.json
```json
{
    "alphabetical": false,
    "borderNone": false,
    "brackets": false,
    "colons": false,
    "commaSpace": false,
    "commentSpace": false,
    "cssLiteral": false,
    "depthLimit": false,
    "duplicates": true,
    "efficient": false,
    "enforceBlockStyle": false,
    "enforceVarStyle": false,
    "extendPref": false,
    "globalDupe": false,
    "indentSpaces": false,
    "leadingZero": false,
    "maxWarnings": false,
    "maxWarningsKill": false,
    "mixed": false,
    "namingConvention": false,
    "parenSpace": false,
    "placeholders": false,
    "semicolons": false,
    "Whitespace": false,
    "universal": false,
    "valid": false,
    "zeroUnits": false,
    "zIndexDuplicates": false,
    "zIndexNormalize": false
}
```

Then execute this:
```
$ stylint site.styl -c config.json
Warning:  duplicate property or selector, consider merging
File: site.styl
Line: 2: @require 'otherfile';
```

**Comment by rossPatton** [comment]
this should be fixed in the latest version, 0.8.12
